### ISSUE TITLE
ADA-XBT is not active on BitMEX

### ISSUE THREAD

**Issue by poshl000** [BitcoinExchangeFH/BitcoinExchangeFH#143]
* Bitcoin exchange feedhandler version: Bump version 2019.1.2
* Python version: /python3.6/
* Operating System: Ubuntu 18.04 LTS

### Description

When I add to configuration.yaml any pair from Bitmex raises an error 'XXX-XBT is not active on BitMEX'

### Configuration file content:

```
    Bitmex:
        instruments:
          - ADAH19
```
### Console output:
```
  File "/home/isset/.local/lib/python3.6/site-packages/cryptofeed/bitmex/bitmex.py", line 34, in __init__
    raise ValueError("{} is not active on BitMEX".format(pair))
ValueError: ADA-XBT is not active on BitMEX
```
### File /cryptofeed/bitmex/bitmex.py contents, with my debug comments:
```
def __init__(self, pairs=None, channels=None, callbacks=None, **kwargs):
        super().__init__('wss://www.bitmex.com/realtime', pairs=None, channels=channels, callbacks=callbacks, **kwargs)
        active_pairs = self.get_active_symbols() //!!DEBUG!! print(active_pairs): ['XRPH19', 'BCHH19', 'ADAH19'...
        for pair in pairs:   //!!DEBUG!! print(pairs): ['ADA-XBT'] -- Why here ADA-XBT? In my configuration file i've referred to ADAH19 
            if pair not in active_pairs:
                raise ValueError("{} is not active on BitMEX".format(pair)) //!!DEBUG!! Raises error, because 'ADA-XBT' not in active_pairs...
        self.pairs = pairs
```

**Comment by poshl000** [comment]
Any other pairs ( XRPH19, BCHH19 etc.) also converts to XRP-XBT, BCH-XBT and raises error ' is not active on BitMEX'

**Comment by gavincyi** [comment]
Thanks for spotting that.

The issue was the feed handler tried to normalize the symbol name for cryptofeed while BitMEX should be excluded from it. It was fixed and please install `BitcoinExchangeFH==2019.1.3`.
